### PR TITLE
#119 Fix sidebar duplication on window resize

### PR DESCRIPTION
**Pull request by oubchid**
Demo:

https://streamable.com/dqi9b8

### CODE REVIEW

File: src/linagora.esn.unifiedinbox/app/components/sidebar/attachment-button/sidebar-attachment-button.pug [hunk 1]
```diff
@@ -1,6 +1,6 @@
 button.btn.btn-link.btn-icon.visible-xl.waves-notransition.active(ng-if="$root.inbox.rightSidebar.isVisible", ui-sref="^", title="{{ 'Hide emails with attachments' | translate }}")
   i.mdi.mdi-paperclip
-button.btn.btn-link.btn-icon.visible-xl.waves-notransition(ng-if="!$root.inbox.rightSidebar.isVisible", ui-sref=".attachments", title="{{ 'Show emails with attachments' | translate }}")
+button.btn.btn-link.btn-icon.visible-xl.waves-notransition(ng-if="!$root.inbox.rightSidebar.isVisible", title="{{ 'Show emails with attachments' | translate }}", contextual-sidebar, data-content-template="/unifiedinbox/views/components/sidebar/attachment/sidebar-attachment.html", data-placement="right", data-controller="inboxListSidebarAttachmentController", data-controller-as="ctrl")
```
Comment (fabienmoyon) [inline src/linagora.esn.unifiedinbox/app/components/sidebar/attachment-button/sidebar-attachment-button.pug:3]: Now we need to display the sidebar in the right if it's a medium and xl device and like the current behavior when it's a small device. You can check how it's done in `Calendar planing view` 


Peek 04-11-2020 08-58

File: src/linagora.esn.unifiedinbox/app/components/sidebar/attachment-button/sidebar-attachment-button.pug [hunk 2]
```diff
@@ -1,6 +1,6 @@
 button.btn.btn-link.btn-icon.visible-xl.waves-notransition.active(ng-if="$root.inbox.rightSidebar.isVisible", ui-sref="^", title="{{ 'Hide emails with attachments' | translate }}")
   i.mdi.mdi-paperclip
-button.btn.btn-link.btn-icon.visible-xl.waves-notransition(ng-if="!$root.inbox.rightSidebar.isVisible", ui-sref=".attachments", title="{{ 'Show emails with attachments' | translate }}")
```
Comment (MichaelBailly) [inline src/linagora.esn.unifiedinbox/app/components/sidebar/attachment-button/sidebar-attachment-button.pug:3]: That means that nowhere in the SPA we use the .attachments route anymore, so you should remove it from the router definition

File: src/linagora.esn.unifiedinbox/app/components/sidebar/attachment-button/sidebar-attachment-button.pug (hunk 1 above)
Comment (MichaelBailly) [inline src/linagora.esn.unifiedinbox/app/components/sidebar/attachment-button/sidebar-attachment-button.pug:3]: Can you try to keep only one button ?

File: src/linagora.esn.unifiedinbox/app/components/sidebar/attachment-button/sidebar-attachment-button.pug [hunk 3]
```diff
@@ -1,6 +1,6 @@
 button.btn.btn-link.btn-icon.visible-xl.waves-notransition.active(ng-if="$root.inbox.rightSidebar.isVisible", ui-sref="^", title="{{ 'Hide emails with attachments' | translate }}")
```
Comment (MichaelBailly) [inline src/linagora.esn.unifiedinbox/app/components/sidebar/attachment-button/sidebar-attachment-button.pug:1]: ui-sref='^' : that won't work I guess.

**Review by MichaelBailly — CHANGES_REQUESTED** [review]

File: src/linagora.esn.unifiedinbox/app/components/sidebar/attachment-button/sidebar-attachment-button.pug (hunk 2 above)
Comment (oubchid) [reply src/linagora.esn.unifiedinbox/app/components/sidebar/attachment-button/sidebar-attachment-button.pug:3]: Yes, I will remove it

**Review by fabienmoyon — APPROVED** [review]

**Comment by oubchid** [comment]
As discussed with Fabien we can keep attachments routes, and fix this issue using CSS

**Pull request closed without merging**